Client/Blogger/src/App.jsx

```diff
@@ -0,0 +1,43 @@
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { ToastContainer, toast } from "react-toastify";
+import { Container, Nav } from "react-bootstrap";
+import HomePage from "./pages/Home";
+import BlogPage from "./pages/BlogPage";
+import NavBar from "./Components/NavBar";
+import FormPage from "./pages/FormPage";
+import { Provider } from "react-redux";
+import reduxStore from "./Redux/store";
+import CreateBlog from "./pages/CreateBlog";
+import EditBlogPage from "./pages/EditBlogPage";
+import "react-toastify/dist/ReactToastify.css";
+
```

Comment:
Why is there additional line spacing between the import statements? Try to maintain consistency in coding to improve readability and prevent overly long files